docs/DatabaseCatalogAPI.yaml

```diff
@@ -683,20 +683,22 @@ components:
       schema:
         type: string
         example: Los Angeles
-    downloaded_at_gte:
-      name: downloaded_at_gte
+    downloaded_after:
+      name: downloaded_after
       in: query
-      description: Filter feed datasets with downloaded date greater or equal to given date.
+      description: Filter feed datasets with downloaded date greater or equal to given date. Date should be in ISO 8601 format.
```

Comment:
Strictly speaking it's a date-time, not just a date. Maybe we could use ISO 8601 Timestamp?

Reply:
What about:  _ISO 8601 Date-Time_